### ISSUE TITLE
Email sender avatars

### ISSUE THREAD

**Issue by maicol07** [bkaankose/Wino-Mail#100]
Hi,
it can be nice to add the avatars of email senders in the messages list and the email details. Maybe the avatars can be grabbed from Gravatar and/or email provider/website favicon (if gravatar is not available)

**Comment by bkaankose** [comment]
This might impact the performance and network usage for no reason. I'd like to implement a 'profile picture' for the account owner for Gmail and Outlook users though.

**Comment by maicol07** [comment]
Yeah, maybe the favicon is a bit too much, but Gravatar is made for email profile pictures.

**Comment by bkaankose** [comment]
Sender avatars are available in the recent builds including the Store version.

Closing as fixed.
